src/main/java/duke/exception/NoCommandException.java

```diff
@@ -0,0 +1,7 @@
+package duke.exception;
```

Comment:
I like how you created multiple exceptions for different situations.

Reply:
Thanks!